src/Hamiltonians/HOCartesianCentralImpurity.jl

```diff
@@ -0,0 +1,200 @@
+"""
+    log_abs_oscillator_zero(n)
+
+Compute the logarithm of the absolute value of the ``n^\\mathrm{th}`` 1D 
+harmonic oscillator function evaluated at the origin. The overall sign is
+determined when the matrix element is evaluated in [`ho_delta_potential`](@ref).
+"""
+function log_abs_oscillator_zero(n)
+    isodd(n) && return -Inf, 1.0
+    x, _ = SpecialFunctions.logabsgamma((1-n)/2)
+    y, _ = SpecialFunctions.logabsgamma(n+1)
+    result = log(pi)/4 + (n/2)*log(2) - x - y/2
+    return result
+end
+
+"""
+    ho_delta_potential(S, i, j; vals)
```

Comment:
```suggestion
    ho_delta_potential(S, i, j; [vals])
```
`vals` is an optional keyword argument, which can be indicated by the square brackets.